10-knative-source-salesforce/salesforce-source.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Apache Camel Salesforce Source
+#
+apiVersion: sources.knative.dev/v1alpha1
+kind: CamelSource
+metadata:
+  name: camel-salesforce-source
+spec:
+  source:
+    integration:
+      configuration:
+        - type: secret
+          value: salesforce
+    flow:
+      from:
+        uri: salesforce:camelTestTopic
```

Comment:
maybe this should be `salesforce:{{salesforce.topic}}` to make it clear it is a parameter the user should set ?

Reply:
I tried it but the property substitution doesn't seem to happen in uri, it's sending `{{salesforce.topic}}` as it is and get invalid character error.

Reply:
even if quoted ? mind opening an issue on camel k runtime if it does not work ?

Reply:
OK filed it - apache/camel-k-runtime#392